input/fsh/modules/family.fsh

```diff
@@ -0,0 +1,175 @@
+Id: SharedDataModelFamilyStudy
+Title: "Shared Data Model for Family Study"
+Description: "The **Shared Data Model for Family Study**"
+* familyID 1..1 string "Unique family study identifier. System identifier used for internal references."
+* externalID 0..* string "Family study identifier. System identifier used for external references."
+* familyType 0..1 code "Describes the 'type' of family study, eg, trio. "
+* description 0..1 string "Free text describing the family study, such as potential inheritance or details about consanguinity"
+* consanguinity 0..1 code "Is there known or suspected consanguinity in this family study?"
+* familyStudyFocus 0..1 code "What is this family study investigating? EG, a specific condition"
+
+CodeSystem: NcpiFamilyTypes
+Id: ncpi-family-types
+Title: "NCPI Family Types CodeSystem"
+Description: "CodeSystem for Types of Families"
+* ^url = $ncpi-family-types
+* ^status = #active
+* ^experimental = false
+* ^publisher = "NCPI FHIR Works"
+* ^content = #fragment
+* #Control-only "Control Only"
+* #Duo "Duo"
+* #Other "Other"
+* #Proband-only "Proband Only"
+* #Trio "Trio"
+* #Trio+ "Trio+"
+
+ValueSet: NcpiFamilyTypesVS
+Id: ncpi-family-types-vs
+Title: "Family Types Codes"
+Description: "A value set with all codes used for the expected family types."
+* ^version = "0.1.0"
+* ^status = #active
+* include codes from system $ncpi-family-types
+
+Extension: FamilyType
+Id: family-type
+Title: "Family Type Extension"
+Description: "Extension containing Family Type"
+* value[x] only CodeableConcept 
+* valueCodeableConcept ^short = "Describes the 'type' of family study, eg, trio."
+* valueCodeableConcept from ncpi-family-types-vs (extensible)
+
+Extension: Description
+Id: description
+Title: "Family Study Description"
+Description: "Free text describing the family study, such as potential inheritance or details about consanguinity"
+* value[x] only markdown 
+* valueMarkdown 0..1
+
+ValueSet: ConsanguinityAssertionVS
+Id: consanguinity-assertion-vs
+Title: "Consanguinity Value Codes"
+Description: "List of codes indicates the level of known consanguinity (blood relation) within a family study."
+* ^experimental = false
+* $snomedct_us#428263003  "NOT suspected"
+* $snomedct_us#415684004  "Suspected"
+* $snomedct_us#410515003  "Known present"
+* $snomedct_us#261665006  "Unknown"
+
+Extension: Consanguinity
+Id: consanguinity
+Title: "Consanguinity Extension"
+Description: "Extension containing Consanguinity"
+* value[x] only CodeableConcept
+* valueCodeableConcept from consanguinity-assertion-vs (extensible)
+* valueCodeableConcept ^short = "Is there known or suspected consanguinity in this family study?"
+
+Extension: FamilyStudyFocus
+Id: family-study-focus
+Title: "Family Study Focus Extension"
+Description: "Extension containing Family Study Focus"
+* value[x] only CodeableConcept 
+* valueCodeableConcept ^short = "What is this family study investigating? EG, a specific condition"
+
+Profile: NcpiFamilyStudy
```

Comment:
I feel like Study and Family should be reversed. So, NcpiStudyFamily. The other way makes it sound like the family itself is the focus of the study.